apps/backend/src/modules/notifications/notification.service.ts

```diff
@@ -77,6 +77,18 @@ class NotificationService implements Service {
 		return await this.notificationRepository.delete(notificationId);
 	}
 
+	public async deleteAllNotificationsByUserId(
+		id: number,
+		userId: number,
+		type: string,
+	): Promise<boolean> {
+		return await this.notificationRepository.deleteAllNotificationByParameters(
```

Comment:
```suggestion
		return await this.notificationRepository.deleteAllNotificationsByParameters(
```
Now in some places you have Notification, in some places Notifications